### ISSUE TITLE
Dynamic styles dont use camel case default 

### ISSUE THREAD

**Issue by eskimoblood** [cssinjs/react-jss#84]
I have the following style: 
```
  box: {
    fontSize: ({ theme }: { theme: Theme }) => theme.font.size.form,
    height: 60,
    padding: 0,
    borderRight: 'solid 1px #fff',
  }
```
while `borderRight` is converted into `border-right`, `fontSize` is not converted in the resulting class:
```
.box-0-25 {
    fontSize: 14px;
}

.box-0-2 {
    height: 60px;
    padding: 0;
    border-right: solid 1px #fff;
}
```

**Comment by kof** [comment]
yes https://github.com/cssinjs/jss/issues/441

**Comment by kof** [comment]
Will keep it open because I have got this issue in this repo already multiple times.

**Comment by kof** [comment]
Fixed, check out react-jss 6.0
